### PR TITLE
added _data dir

### PR DESCRIPTION
**Pull request by aleesteele**
#28 - Made data dir with content types, made all filterable & without icons.

### CODE REVIEW

**Comment by aleesteele** [comment]
Changes have been added - except for `news`, which I kept title_single (semantics, can be made into title_plural).

**Comment by jnschrag** [comment]
@aleesteele All of the types need to have `title_single` and `title_plural` even if the values of them are actually the same. So please update the `news` type accordingly.

**Comment by aleesteele** [comment]
`title_plural` has been added to `news`.

File: _data/content-types.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+- type: commentary
+  title_single: Commentary
+  title_plural: Commentaries
+  filterable: true
+- type: event
+  title_single: Event
+  title_plural: Events
+  filterable: true
+- type: documentary
+  title_single: documentary
```
Comment (jnschrag) [inline _data/content-types.yml]: Can you capitalize "documentary" please?

**Comment by aleesteele** [comment]
Documentaries has been removed.

**Pull request closed without merging**